apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.html

```diff
@@ -1,6 +1,9 @@
 <daff-sidebar-viewport (backdropClicked)="close()" [mode]="mode$ | async" [opened]="showSidebar$ | async">
   <daff-sidebar class="daffio-sidebar" (escapePressed)="close()">
     <button type="button" daff-button class="daffio-sidebar__button">Get Started</button>
-    </daff-sidebar>
+    Guides
```

Comment:
Remove this word entirely for now.

Reply:
Removed 9887de41132ffb5e64d10145ef9642545ad5b4f7